Formula/alerter.rb

```diff
@@ -0,0 +1,15 @@
+class Alerter < Formula
+  desc "Send User Alert Notification on Mac OS X from the command-line"
+  homepage "https://github.com/vjeantet/alerter"
+  url "https://github.com/vjeantet/alerter/archive/refs/tags/004.tar.gz"
+  version "4"
+  sha256 "c4c16735e1a57ce04a5acfa762bd44ab8ef10884308725ab886b984b1de17bfe"
+  license "MIT"
+
+  def install
```

Comment:
```suggestion
  def install
    xcodebuild "-project", "alerter.xcodeproj", "-target", "alerter", "-configuration", "Release", "SYMROOT=build"
    bin.install "build/Release/alerter"
```
You probably need something like this. You should also have a `depends_on xcode: :build`.

Reply:
Thanks for this. I'm getting some signing error, need time to change the signing settings to manual or something